src/webserver/Firebasey.jl

```diff
@@ -285,17 +285,17 @@ function diff(o1, o2)
 end
 
 # ╔═╡ b8c58aa4-c24d-48a3-b2a8-7c01d50a3349
-function diff(o1::Nothing, o2)
+function diff(o1::Union{Nothing, Missing}, o2)
```

Comment:
Since `cell_execution_order`, `referenced_cells`, `dependent_cells`, but also `runtime`(!) start as _missing_, we need diff to handle that.
Otherwise it's `TypeError: non-boolean (Missing) used in boolean context`.

Reply:
Good finding! Maybe it makes sense to correct this in a separate PR?

Reply:
Yes - Let me write something up so Fons and Michiel can also share thoughts!

Reply:
Why did that not give errors before?

Reply:
In Dynamic.jl, runtime is converted to nothing if missing:
`                "runtime" => ismissing(cell.runtime) ? nothing : cell.runtime,
`
Maybe this is the reason?